src/plugins/orangehrmCorporateDirectoryPlugin/Api/EmployeeDirectoryAPI.php

```diff
@@ -56,6 +56,37 @@ class EmployeeDirectoryAPI extends Endpoint implements CrudEndpoint
     ];
 
     /**
+     * @OA\Get(
+     *     path="/api/v2/directory/employees/{empNumber}",
+     *     tags={"Directory"},
```

Comment:
missing `@OA\PathParameter`
```suggestion
     *     tags={"Directory"},
     *     @OA\PathParameter(
     *         name="empNumber",
     *         @OA\Schema(type="integer")
     *     ),
```